app/assets/javascripts/ui_models/app_state/preferences_state.ts

```diff
@@ -0,0 +1,26 @@
+import { action, computed, makeObservable, observable } from 'mobx';
+
+export class PreferencesState {
+  private _open = true;
```

Comment:
We could make this one public and get rid of the getter

Reply:
I wanted a more verby API to the property :) 
I adjusted the method names according to your suggestion below  :)